app/styles/partials/utils.scss

```diff
@@ -73,14 +73,18 @@
   padding-right: .25rem !important;
 }
 
-.pr-2 {
```

Comment:
Why removing this?

Reply:
sorry, got confused with pl-2, pl-2 is unused css